packages/Text/src/styles.ts

```diff
@@ -11,6 +11,15 @@ const MOBILE_VARIANTS = {
   h5: 'h6',
   h6: 'h6',
 }
+const headingsLineHeightFixer = {
+  h0: 0.825,
+  h1: 0.75,
+  h2: 0.825,
+  h3: 1.075,
+  h4: 1,
+  h5: 0.9,
+  h6: 0.9,
+}
```

Comment:
Maybe we can add all of this on typography core theme ?

Reply:
I just wanted to keep all the ugly fixes in the same place until we have a correct font 🫠